README.md

```diff
@@ -56,13 +56,32 @@ cf set-env postgres-tinsmith SB_BROKER_PASSWORD a-secret
 
 # start the app
 cf start postgres-tinsmith
+```
+
+Then, create the Tinsmith service broker. If you have the administrator role:
 
-# register this tinsmith as a service broker in CF
+```shell
 cf create-service-broker postgres-tinsmith my-broker a-secret \
   https://postgres-tinsmith.$APP_DOMAIN
+```
+
+If you only have the space developer role:
+
+```shell
+# make sure to target your space
+cf target -s <SPACE>
+
+cf create-service-broker postgres-tinsmith my-broker a-secret \
+  https://postgres-tinsmith.$APP_DOMAIN --space-scoped
+```  
+
+Lastly, enable the new `postgres` service provided by this Tinsmith and verify it:
+
+```shell
+# enable the service
```

Comment:
Enabling the service is something that administrators do for non-space-scoped brokers, so this line belongs with the administrator command above.